### DISCUSSION TITLE
Is there any way to load the css for this on a single page?

### DISCUSSION THREAD

**Discussion by the1gofer** [efemkay/obsidian-modular-css-layout#74] [General]
I really want to use this CSS, but it slows down live preview to a crawl on all my pages.  Is there any way to load this on a page by page basis?

**Comment by efemkay** [comment]
it would be difficult for me to make it vault wide and per page basis at the same time. i've updated parts of the snippets to improve the css performance. perhaps you can update it and see if that solves it. i'm still working some other parts of the css to make it more performant.
